src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -729,6 +721,10 @@ std::string postUpdateContext
       // If some CER (but not all) fails, then it is a partial update
       parseDataP->upcrs.res.oe.fill(SccContextElementNotFound, "Attributes that were not updated: { " + failing + " }", "PartialUpdate");
     }
+    else {  // fails == 0
```

Comment:
style guide ... ( `{` on a separate line)

Reply:
Fixed in 5a3ae39